### PR TITLE
Update governance documents

### PR DESCRIPTION
**Pull request by zjcnb**
For #12065

### CODE REVIEW

File: docs/document/content/user-manual/shardingsphere-jdbc/usage/governance/yaml.cn.md [hunk 1]
```diff
@@ -32,28 +32,30 @@ weight = 2
...
+// 创建 ShardingSphereDataSource
+DataSource dataSource = YamlShardingSphereDataSourceFactory.createDataSource(yamlFile);
 ```
 
-## 使用 GovernanceShardingSphereDataSource
+## 使用 ShardingSphereDataSource
 
-通过 YamlGovernanceShardingSphereDataSourceFactory 工厂创建的 GovernanceShardingSphereDataSource 实现自 JDBC 的标准接口 DataSource。
+通过 YamlShardingSphereDataSourceFactory 工厂创建的 ShardingSphereDataSource 实现自 JDBC 的标准接口 DataSource。
 可通过 DataSource 选择使用原生 JDBC，或JPA， MyBatis 等 ORM 框架。
```
Comment (taojintianxia) [inline docs/document/content/user-manual/shardingsphere-jdbc/usage/governance/yaml.cn.md:53]: it's not necessary add a whitespace between English Charactor and a `，`

Comment (taojintianxia) [inline docs/document/content/user-manual/shardingsphere-jdbc/usage/governance/yaml.cn.md:53]: `或JPA` change to `或 JPA`

Comment (zjcnb) [reply docs/document/content/user-manual/shardingsphere-jdbc/usage/governance/yaml.cn.md:53]: yes, i will modify it.

Comment (zjcnb) [reply docs/document/content/user-manual/shardingsphere-jdbc/usage/governance/yaml.cn.md:53]: ok

**Pull request closed without merging**